google/ggrc-core
# GGRC-8782 People list is updated only after page refresh

# Issue description

When Persons list was changed w/o changing counts of people Person tab should be reload.

# Steps to test the changes

1. Open any Program
2. Edit program
3. Change Person in any role field on another Person
4. Save changes
5. Open People tab
6. Expected: New added user is displayed in the People tab

# Solution description

Added Person list ids check. Person tab should be reload, if ids of showed users doesn't equal ids from access list.

# Sanity checklist




# PR Review checklist
